### DISCUSSION TITLE
slack链接失效了

### DISCUSSION THREAD

**Discussion by wniter** [quanxiang-cloud/quanxiang#48] [Q&A]
slack连接失效了，

**Comment by kevinlee607** [comment]
我试了下，是可以的哦。有截图吗？

**Reply by wniter** [reply]
![image](https://user-images.githubusercontent.com/47174623/182368317-94933af9-ea5d-4205-9477-54c9302c7f69.png)

**Reply by wniter** [reply]
已解决
